OTAcceleratorCore.xcodeproj/project.pbxproj

```diff
@@ -166,12 +175,33 @@
 			isa = PBXFrameworksBuildPhase;
 			buildActionMask = 2147483647;
 			files = (
+				E33A3600D0F40630ED4C88C9 /* libPods-OTAcceleratorCoreTests.a in Frameworks */,
```

Comment:
I think it is a good practice to run `pod deintegrate` before checking the project file.

This way we remove pod reference in the project file

Reply:
Thank you for the suggestion @robjperez! Fixed